tests/conftest.py

```diff
@@ -111,9 +111,9 @@
 asyncio.set_event_loop_policy = lambda policy: None
 
 
-def pytest_addoption(parser: pytest.Parser) -> None:
-    """Register custom pytest options."""
-    parser.addoption("--dburl", action="store", default="sqlite://")
+#def pytest_addoption(parser: pytest.Parser) -> None:
+#    """Register custom pytest options."""
+#    parser.addoption("--dburl", action="store", default="sqlite://")
```

Comment:
Why is this commented out?

Reply:
Oops, this was committed by mistake. Pytest does not like it on my machine for some reason. Thanks!